### ISSUE TITLE
how to test？

### ISSUE THREAD

**Issue by igotsmoke9** [xindongzhang/ELAN#19]

**Comment by zhang13359** [comment]
tensor = sr.squeeze(0)
tensor = tensor.detach().cpu().numpy().transpose(1, 2, 0)
image = cv2.imwrite(sr_file, tensor)
